talktorials/11_online_apis/11b_online_docking.ipynb

```diff
@@ -0,0 +1,927 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Practice\n",
+    "\n",
+    "\n",
+    "### Get files from part A\n",
+    "\n",
+    "First, we will define some variables pointing to the files we obtained in part A:\n",
+    "\n",
+    "- `PROTEIN`: path to the `.mol2` file containing the structure of the target, without any ligands or ions\n",
+    "- `COMPLEX`: path to the `.pdb` file containing the structure of the target AND the native ligand in its binding site\n",
+    "- `SMILES_FILE`: path to the `.txt` file containing the SMILES representations for all the similar compounds found in PubChem\n",
+    "- `smiles`: list of the SMILES string contained in `SMILES_FILE`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "PROTEIN = \"data/protein.mol2\"\n",
+    "COMPLEX = \"data/complex.pdb\"\n",
+    "SMILES_FILE = \"data/similar_smiles.txt\"\n",
+    "with open(SMILES_FILE) as f:\n",
+    "    smiles = [line.strip() for line in f]"
+   ]
```

Comment:
In order to run the current `step_03_swissdock()` (maybe users would like to try and check if the server is still down), the `protein` needs to be loaded.